vendor.conf

```diff
@@ -6,7 +6,7 @@ github.com/containernetworking/cni v0.7.0-alpha1
 github.com/containers/image 93bced01015eb94bec4821df1876314be8197680
 github.com/boltdb/bolt v1.3.1
 github.com/containers/libpod v1.0
-github.com/containers/storage 06b6c2e4cf254f5922a79da058c94ac2a65bb92f
+github.com/containers/storage 0a30cf16081e9a4c4bc0d2571c29c7f5201f06a5
```

Comment:
Can you pull via v1.10 instead of UUID?

Reply:
Apparently I can use v1.10 instead, just changed, repushing shortly.